src/components/accordionItem/AccordionItem.jsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import { Dropdown } from '../index';
+import * as S from './AccordionItem.styles';
+
+const AccordionItem = ({ data, toggle, clicked }) => {
+  return (
+    <React.Fragment>
```

Comment:
FYI: React.Fragment has shortened version, you can use both options

Reply:
Ok, I fixed to `<></>`.